src/components/PersonLInk.tsx

```diff
@@ -0,0 +1,79 @@
+import { Link, useParams, useSearchParams } from 'react-router-dom';
+import cn from 'classnames';
+
+import { useMemo } from 'react';
+import { Person } from '../types';
+
+type Props = {
+  person: Person;
+  filteredPeople: Person[];
+};
+
+export const PersonLink: React.FC<Props> = ({ person, filteredPeople }) => {
+  const { born, died, fatherName, motherName, name, sex, slug } = person;
+
+  const { slugParam } = useParams();
+  const [searchParams] = useSearchParams();
+
+  const hasMother = useMemo(
```

Comment:
```suggestion
  const maybeMother = useMemo(
```
`has` sounds like it's boolean value